vite.config.ts

```diff
@@ -18,13 +18,24 @@ export default defineConfig(({ mode }) => {
 
   return {
     build: {
-      chunkSizeWarningLimit: 900,
       outDir,
       rollupOptions: {
         input: {
-          main: resolve(__dirname, 'index.html'),
+          index: resolve(__dirname, 'index.html'),
           _scrivito_extensions: resolve(__dirname, '_scrivito_extensions.html'),
         },
+        output: {
+          manualChunks: {
+            react: [
```

Comment:
Editing no longer works 😿 . How to reproduce:

* Visit https://edit.scrivito.com/d0a154d76edf2a7bd991fc658e700a1d~https://dave-default-chunk-size-limi.scrivito-portal-app.pages.dev/en
* Switch to edit mode.
* Open the widget properties e.g. of the Columns Widget or an icon widget

Expected:
The column/icon editor is shown and usable.

Actual:
No editor is shown and the js console prints the following:

```
Uncaught Error: Minified React error #299; visit https://reactjs.org/docs/error-decoder.html?invariant=299 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Rg.gt.createRoot (react-Drplhdjg.js:40:55773)
    at Pt (index-BHeeQ0Eb.js:15:3960)
    at index-BHeeQ0Eb.js:15:3938
```